maintenance-filter-controller/src/main/java/org/ccci/maintenance/MaintenanceControlClient.java

```diff
@@ -86,9 +91,54 @@ public Failure(String message, IOException exception)
             }
             log.info("successfully updated window on " + server);
         }
+        if (failures.isEmpty())
+        {
+            printConfirmationOfWindowBoundaries(window);
+        }
         return failures;
     }
 
+    /** this helps the user verify that he got the timestamps correct in his config file*/
+    private void printConfirmationOfWindowBoundaries(MaintenanceWindow window)
+    {
+        DateTime now = new DateTime();
+        String beginPhrase = getRelativeTimePhrase(
+            now,
+            window.getBeginAt(),
+            "began",
+            "will begin");
+        String endPhrase = getRelativeTimePhrase(
+            now,
+            window.getEndAt(),
+            "ended",
+            "will end");
+
+        log.info("window " + beginPhrase + " and " + endPhrase);
+    }
+
+    private String getRelativeTimePhrase(
+        DateTime now,
+        DateTime begin,
+        String pastVerb,
+        String futureVerb)
+    {
+        PeriodType type = PeriodType.standard().withMillisRemoved();
+        Period period;
+        String verb;
+        if (now.isBefore(begin))
+        {
+            period = new Period(now, begin, type);
+            verb = futureVerb + " in %s";
+        }
+        else
+        {
+            period = new Period(begin, now, type);
+            verb = pastVerb + " %s ago";
+        }
+        PeriodFormatter periodFormatter = PeriodFormat.getDefault();
+        return String.format(verb, periodFormatter.print(period));
```

Comment:
the thing that makes that hard is that in one case you put the formatted period at the end of the string, and in the other you put it at the beginning.

I guess I could rearrange this to use just concatenation, but it'd be more cluttered.